src/viewProviders/schemas.test.ts

```diff
@@ -18,12 +23,134 @@ describe("SchemasViewProvider methods", () => {
   });
 
   it("getTreeItem() should pass ContainerTreeItems through directly", () => {
+    const container = new ContainerTreeItem<Schema>("test", TreeItemCollapsibleState.Collapsed, [
+      TEST_CCLOUD_SCHEMA,
+    ]);
+    const treeItem = provider.getTreeItem(container);
+    assert.deepStrictEqual(treeItem, container);
+  });
+});
```

Comment:
Functionally unchanged, just updated imports so we could use `TreeItemCollapsibleState` instead of `vscode. TreeItemCollapsibleState`